apps/challenges/models.py

```diff
@@ -12,6 +14,11 @@
 class Challenge(TimeStampedModel):
 
     """Model representing a hosted Challenge"""
+
+    def __init__(self, *args, **kwargs):
+        super(Challenge, self).__init__(*args, **kwargs)
+        self.__original_evaluation_script = self.evaluation_script
```

Comment:
I think we are good with making it `semiprivate` instead of making it `superprivate`. So, we would go with single underscore.